docs/learn/learn-auction.md

```diff
@@ -272,6 +272,50 @@ less. Together Dave's and Emily's bids only equals a valuation of `560`.
 Charlie's valuation for the entire range is `600`. Therefore Charlie is awarded the complete range
 of the parachain slot.
 
+## Parachain Lease Extension
+
+Before the slot lease expires, parachains have to bid and win another auction for continuity of the
+lease. To avoid any downtime in connectivity and minimize the risk of losing a subsequent auction, 
+parachain teams need to plan well ahead to bid on and secure an adjacent slot before their current 
+lease period ends. Explained in the section above, each slot auction lets you bid for 8 LPs
+(Lease Periods) which enables two scenarios for the parachain's lease extension -
+
+### Lease Extension with Overlapping Slots
+
+Aquire a slot where the first LP is before the last LP of the current slot.
+
+- Register a new 'paraId'
+- Win a slot auction with the new 'paraId'
+
+The parachain team will have access to two slots: one that will end soon, and one that just started.
+Both slots will have at least one LP in common. When the old slot transitions to their last LP, 
+the parachain can [swap](https://github.com/paritytech/polkadot/pull/4772) the slots. This can be 
+done via [on-chain governance](https://kusama.polkassembly.io/post/1491). The 'swap'
+call is available in the 'registrar' pallet. 
+
+![Parachain Slot Swap](../assets/para-swap.png)
+
+It is also possible to perform slot swap via XCM.
```

Comment:
I think this sentence belongs to the next paragraph, "The slot swap via XCM.."

Reply:
Yeah, seems redundant. I'll delete this line